src/test/java/com/mathworks/ci/RunMatlabBuildStepTest.java

```diff
@@ -123,6 +123,31 @@ public void verifyStartupOptionsSameAsScript() throws Exception {
         j.assertLogContains("-nojvm -uniqueoption", build);
     }
 
+    /*
+     * Verify appropriate build options are invoked as in pipeline script
+     */
+    @Test
+    public void verifyBuildOptionsSameAsScript() throws Exception {
+        project.setDefinition(
+                new CpsFlowDefinition("node { runMATLABBuild(buildOptions: '-continueOnFailure -skip compile') }", true));
+
+        WorkflowRun build = project.scheduleBuild2(0).get();
+        j.assertLogContains("-continueOnFailure -skip compile", build);
+    }
+
+    /*
+     * Verify appropriate tasks and build options are invoked as in pipeline script
+     */
+    @Test
+    public void verifyTasksAndBuildOptionsSameAsScript() throws Exception {
+        project.setDefinition(
+                new CpsFlowDefinition("node { runMATLABBuild(tasks: 'compile', buildOptions: '-continueOnFailure -skip test') }", true));
+
+        WorkflowRun build = project.scheduleBuild2(0).get();
+        j.assertLogContains("compile", build);
+        j.assertLogContains("-continueOnFailure -skip test", build);
+    }
```

Comment:
Nitpick: I'm not sure that this test case adds value if we already have tests covering both task specification and build option specification.